docs/getting-started.md

```diff
@@ -112,18 +125,8 @@ availability. For example, here is an example `package.json`:
 }
 ```
 
-Notes:
-- While git tags are signed, `npm` will not check the signature
-of the git tag.
-- See [Git URLs as Dependencies][giturls] `npm` documentation for
-additional details for using git as a dependency.
-
-If your project shares any dependencies you may want to de-duplicate, you can
-do this by running:
-
-```sh
-npm dedupe
```

Comment:
Maybe add `dedupe` notes back for now. Or wait for gpk updates.

Reply:
Yeah, gpk can be updated in this regard, likely with a `gpk install --dedupe` option or something similar in regards to bundled dependencies.

Reply:
Can by tracked with https://github.com/braydonf/gpk/issues/21